environment-dev.yml

```diff
@@ -5,7 +5,7 @@ dependencies:
     - python
     # required
     - fiona>=1.8
-    - pandas>=1.0.0
+    - pandas[test]>=1.0.0
```

Comment:
This won't work, as this file is for conda (which doesn't have the concept of extras)

Reply:
Thanks for the feedback! I don’t have any experience with conda packaging. I’ll drop the `environment-dev.yml` change.